auditwheel/patcher.py

```diff
@@ -64,3 +82,75 @@ def set_rpath(self,
         check_call(['patchelf', '--remove-rpath', file_name])
         check_call(['patchelf', '--force-rpath', '--set-rpath',
                     rpath, file_name])
+
+    def get_rpath(self,
+                  file_name: str) -> str:
+
+        return check_output(['patchelf', '--print-rpath',
+                             file_name]).decode('utf-8').strip()
+
+    def append_rpath(self,
+                     file_name: str,
+                     rpath: str,
+                     wheel_base_dir: str) -> None:
+        """Add a new rpath entry to a file while preserving as many existing
```

Comment:
Ah, good call, ok -- so move this into `repair_wheel()`?

Reply:
Ok, I moved all this into `repair_wheel()` along with the tests. There is a slight change to the way patching works -- the `patcher` calls now always modify the absolute path to the library. I can't think of a reason why this would be a problem but it is different. If you object it would be very easy to change.